avocado/core/parser.py

```diff
@@ -145,3 +142,7 @@ def finish(self):
                     sub.choices[self.args.subcommand].error(msg)
 
             self.application.error(msg)
+
+        # Allow overriding default params by plugins
```

Comment:
Actually I can't tell anymore. I know I changed it for a reason, but maybe that reason disappeared with all the changes I made afterwards. I'll remove this (unless it breaks something in later commits, then I'll move it to that commit).